libs/SalesforceReact/metro.config.js

```diff
@@ -0,0 +1,11 @@
+const {getDefaultConfig, mergeConfig} = require('@react-native/metro-config');
+
+/**
+ * Metro configuration for React Native
```

Comment:
We do have a `start` command in this module's `package.json`, so this new file is required to use that.  However, when would we start this React Native module?  If that's not used, the command and this file could be removed.